app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str, *args) -> None:
+        self.name = name
+        self.skills = [*args]
```

Comment:
remove args, this can lead to unexpected behavior and type annotations are missed